cloudflare/resource_cloudflare_load_balancer_monitor_test.go

```diff
@@ -236,6 +237,7 @@ resource "cloudflare_load_balancer_monitor" "test" {
   path = "/custom"
   interval = 60
   retries = 5
+	port = 8080
```

Comment:
Looks like the indentation is out here. Would you mind aligning it with the other values?

Reply:
Not a problem. Done.